main.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // Version holds the current Gitea version
-const Version = "0.9.99.0915"
+var Version = "v0.9.0+dev"
```

Comment:
`1.0.0+dev` means we have released 1.0 and begin dev.
`0.9.0+dev` means we have released 0.9 and begin dev.